zenith_utils/src/http/mod.rs

```diff
@@ -2,3 +2,7 @@ pub mod endpoint;
 pub mod error;
 pub mod json;
 pub mod request;
+
+/// Current fast way to applly simple http routing in varuious Zenith binaries.
+/// Reexported for sake of uniform approach, that could be later replaced with better alternatives, if needed.
+pub use routerify::{ext::RequestExt, RouterBuilder};
```

Comment:
```suggestion

/// Current fast way to apply simple http routing in various Zenith binaries.
/// Re-exported for sake of uniform approach, that could be later replaced with better alternatives, if needed.
pub use routerify::{ext::RequestExt, RouterBuilder};
```
I don't understand what this is all about though. Why is the re-export needed?

Reply:
@LizardWizzard  and @funbringer most probably can elaborate better than me, since I've followed their advice here :) 

In general, the idea is to put all that we use in http server set-up (routing, tokens, etc.) behind a single crate and use it in all binaries.
This way we get:
* all http-related things in one place (`zenith_utils::http::*`)
* possible ease the dependency upgrades later (well, not for this particular case since we reexport, but in theory)
* same code for all places that can be later replaced with a "more decent" http/routing/whatever library

Reply:
Ok, make sense!